package.json

```diff
@@ -11,11 +11,12 @@
   },
   "devDependencies": {
     "env-cmd": "^10.1.0",
-    "ngrok": "^3.3.0"
+    "ngrok": "^3.3.0",
+    "nodemon": "^2.0.4"
```

Comment:
`yarn.lock` isn't updated. Please add that too. Everything else looks good

Reply:
Resolved